### ISSUE TITLE
Disable back arrow at File Backup after word import

### ISSUE THREAD

**Issue by danimoh** [nimiq/keyguard-next#246]
Otherwise, it's possible to import multiple wallets by words in one import request which is probably not what we want.

**Comment by sisou** [comment]
I think a cleaner approach is fixing the problem, not the symptom. Meaning I'd rather reset the keys array to empty when going from words to password input.

**Comment by sisou** [comment]
I believe this is actually a non-issue since the array is already reset when completing the words input: https://github.com/nimiq/keyguard-next/blob/master/src/request/import/ImportWords.js#L237.

Are you sure this is a working bug, @danimoh?

**Comment by Bettelstab** [comment]
> Are you sure this is a working bug, @danimoh?

xD

**Comment by danimoh** [comment]
Yes, but the wallet is immediately stored to the database with `_storeKeys` after a password was defined.
I believe this is an actual bug.

**Comment by nibhar** [comment]
An option would be to remove the newly created account(s) from the indexedDb again in case the user imports different words before these words are stored (therefore not only overwriting the returned keyInfo but also the key itself).
Another option would be to not store the key until the request actually resolves, which definitely would be a bigger change in the code.

I agree with @sisou that it is probably not very high priority.
